### DISCUSSION TITLE
Open source tools first?

### DISCUSSION THREAD

**Discussion by haydenbleasel** [vercel/next-forge#364] [Polls]
I've had a fair few recommendations on this repo and social to select open source tooling as the defaults for next-forge. For example, BetterAuth over Clerk, OpenStatus over BetterStack, etc. I think the _general_ tradeoff will be around convenience and scalability of third-party tools vs. low pricing and ownership of open source tools.

Not really sold on the transition yet myself, so I'd love to hear your thoughts.

**Comment by roninreilly** [comment]
ofc open-source should be first
